### PR TITLE
nodle parachain bringup

### PR DESCRIPTION
**Pull request by rajesh-nodle**
**Active PR**

nodle parachain bringup

### CODE REVIEW

File: node/build.rs [hunk 1]
```diff
@@ -25,7 +41,7 @@ fn build_shell_completion() {
     }
 }
 
-/// Build the shell auto-completion for a given Shell
+// /// Build the shell auto-completion for a given Shell
```
Comment (ETeissonniere) [inline node/build.rs:44]:
```suggestion
/// Build the shell auto-completion for a given Shell
```

File: runtimes/eden/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        // System
+        System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
+        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 1,
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 2,
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 3,
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 4,
+
+        // Governance
+        TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 5,
+        TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 6,
```
Comment (ETeissonniere) [inline runtimes/eden/src/lib.rs:83]: we can remove these two

Comment (rajesh-nodle) [reply runtimes/eden/src/lib.rs:83]: @ETeissonniere , for me it occurs like, we lose control over addition & removal of allocations oracle members, is it OK ?

```
    #[pallet::storage]
    #[pallet::getter(fn oracles)]
    pub type Oracles<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;

```

Comment (ETeissonniere) [reply runtimes/eden/src/lib.rs:83]: We can use `EnsureRoot` for controlling who can add oracles or remove them

Comment (rajesh-nodle) [reply runtimes/eden/src/lib.rs:83]: Make sense, looks good.

Comment (rajesh-nodle) [reply runtimes/eden/src/lib.rs]: Done @Eliott, added AllocationsOracles back, and its working for sudo origin

**Pull request merged**